packages/core-ethereum/src/index.ts

```diff
@@ -442,8 +462,12 @@ export default class HoprCoreEthereum extends EventEmitter {
         error: err
       }
     }
-
     log('Successfully submitted ticket', ackTicket.response.toHex())
+
+    // bump commitment when on-chain ticket redemption is successful
+    await bumpCommitment(this.db, channelId, commitmentPreImage)
```

Comment:
What happens if we fail earlier already. The ticket should then be removed?

Reply:
So far they (failed with out-of-commits, preimage-is-empty, not-a-winning-ticket) are not removed from the DB. I think it makes sense to treat those acknowledged tickets as a "loss" and removed from the DB.